Add missing include file to get definition of PATH_MAX

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Upstream-Status: Pending

Index: libpciaccess-0.13.3/src/linux_sysfs.c
===================================================================
--- libpciaccess-0.13.3.orig/src/linux_sysfs.c
+++ libpciaccess-0.13.3/src/linux_sysfs.c
@@ -44,6 +44,7 @@
 #include <sys/mman.h>
 #include <dirent.h>
 #include <errno.h>
+#include <limits.h>
 
 #if defined(__i386__) || defined(__x86_64__) || defined(__arm__)
 #include <sys/io.h>
